alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/transport/TransportExecuteMonitorAction.kt

```diff
@@ -39,8 +41,14 @@ class TransportExecuteMonitorAction @Inject constructor(
 ) : HandledTransportAction<ExecuteMonitorRequest, ExecuteMonitorResponse> (
         ExecuteMonitorAction.NAME, transportService, actionFilters, ::ExecuteMonitorRequest) {
 
+    private var user: User? = null
+
     override fun doExecute(task: Task, execMonitorRequest: ExecuteMonitorRequest, actionListener: ActionListener<ExecuteMonitorResponse>) {
 
+        val userStr = client.threadPool().threadContext.getTransient<String>(ConfigConstants.OPENDISTRO_SECURITY_USER_AND_ROLES)
+        log.debug("User and roles string from thread context: $userStr")
```

Comment:
I don't remember if `log.debug()` prepends any context information but if it doesn't, could we make this message a little more specific to know that it's coming from the execute transport action?

Reply:
Double checked, it looks like below: `[2020-12-01T00:10:12,201][DEBUG][c.a.o.a.t.TransportIndexMonitorAction] [38f9d35bb887.ant.amazon.com] User and roles string from thread context: admin|admin|own_index,all_access`

It has the transport class name.

Reply:
Cool, thanks for checking